language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/completions/providers/context/ConstantDeclarationNodeContext.java

```diff
@@ -105,8 +104,8 @@ public boolean onPreValidation(BallerinaCompletionContext context, ConstantDecla
     }
 
     @Override
-    protected List<LSCompletionItem> initializerContextCompletions(BallerinaCompletionContext context,
-                                                                   Node typeDesc, Node initializer) {
+    protected List<LSCompletionItem> initializerContextCompletions(BallerinaCompletionContext context, 
```

Comment:
Shall we remove the extra space after the comma?

Reply:
If we do so the length of the line will be greater than 120 chars.